docs/sources/setup-grafana/configure-security/configure-security-hardening.md

```diff
@@ -0,0 +1,100 @@
+---
+aliases:
+  - /docs/grafana/latest/setup-grafana/configure-security/configure-security-hardening/
+description: Security hardening enables you to apply additional security which might stop certain vulnerabilities from being exploited by a malicious attacker.
+title: Configure security hardening
+---
+
+# Configure security hardening
+
+Security hardening enables you to apply additional security which might stop certain vulnerabilities from being exploited by a malicious attacker.
+
+> **Note:** These settings are available in the [grafana.ini configuration file]({{< relref "../configure-grafana/#configuration-file-location" >}}). To apply changes to the configuration file, restart the Grafana server.
+
+## Additional security for cookies
+
+If Grafana uses HTTPS, it is possible to further secure the cookie that is used as authentication to access the Web UI. By securing the cookie further, you might mitigate against certain attacks where an attacker could get hold of the cookie value.
+
+> **Note:** Grafana must use HTTPS in order for the below configurations to work properly.
+
+### Add a secure attribute to cookies
+
+The secure attribute will add a `Secure` attribute to the cookie that is used to authenticate users and enforces users to only send the cookie over a valid HTTPS secure connection. This configuration provides mitigation against some MITM attacks.
+
+Example:
+
+```toml
+# Set to true if you host Grafana behind HTTPS. The default value is false.
+cookie_secure = true
+```
+
+### Add a SameSite attribute to cookies
+
+Set the _cookie_samesite_ option to `strict`. This will enforce clients to not send the cookie in requests that were made cross-site, but only from the site that sat the cookie. By setting the option to "strict", you will mitigate almost all CSRF-attacks.
```

Comment:
```suggestion
To mitigate almost all CSRF-attacks, set the _cookie_samesite_ option to `strict`. This setting prevents clients from sending the cookie in requests that are made cross-site, but only from the site that sat the cookie.
```

Reply:
^^ I'm tweaked the wording, but not sure what `sat` means. I think it's a typo, but wasn't sure.